docs/source/markdown/podman-create.1.md

```diff
@@ -419,6 +419,16 @@ Logging driver specific options.  Used to set the path to the container log file
 
 `--log-opt path=/var/log/container/mycontainer.json`
 
+**--log-opt**=*tag*
+
+Specify a custom log tag for the container.  For example:
```

Comment:
We should mention that other log options are available but not supported. Maybe "Set custom logging configuration. Presently supports the **tag** option which specified a custom log tag for the container. For example..."